builtin/mainmenu/modmgr.lua

```diff
@@ -286,21 +286,36 @@ end
 function modmgr.get_dependencies(modfolder)
 	local toadd = ""
 	if modfolder ~= nil then
-		local filename = modfolder ..
-					DIR_DELIM .. "depends.txt"
-
-		local dependencyfile = io.open(filename,"r")
-
-		if dependencyfile then
-			local dependency = dependencyfile:read("*l")
-			while dependency do
+		local mod_conf = Settings(modfolder .. DIR_DELIM .. "mod.conf")
+		local dependencies = nil
+		if mod_conf then
+			dependencies = mod_conf:get("depends");
+		end
+		if dependencies then
+			local list = dependencies:split()
```

Comment:
Split splits by default with ",", which should be the separator between mod names.